docs/guides/automating-a-process-using-bpmn.md

```diff
@@ -0,0 +1,103 @@
+---
+id: automating-a-process-using-bpmn
+title: Automating a process using BPMN
+description: A quickstart to how BPMN can automate your most complex business processes using an easy-to-adopt visual modeling language.
+keywords: [workflow, modeling]
+---
+
+<span class="badge badge--beginner">Beginner</span>
+<span class="badge badge--medium">Time estimate: 20 minutes</span>
+
+## What is BPMN?
+
+Business Process Model and Notation (BPMN) is the global standard for process modeling. With BPMN, automate your most complex business processes using an easy-to-adopt visual modeling language.
+
+Processes are the algorithms that determine how an organization runs based on independent tasks. Successful businesses grow from proven, effective processes. Therefore, Camunda’s workflow engine executes processes defined in BPMN.
+
+![empty bpmn diagram sample](./img/automate-any-process-anywhere.png)
```

Comment:
Can you double check with Marketing that we can use this? I know this slogan was a thing, then it wasn't, but I'm not sure.

Reply:
Will follow up, but truthfully, I don't think we need this attachment after a second look, given we outline an example immediately after. Thoughts?